doc/Makefile

```
@@ -37,32 +38,45 @@ help:
 
 # Generate the doxygen xml (for Sphinx) and copy the doxygen html to the
 # api folder for publishing along with the Sphinx-generated API docs.
-
-copy-to-sourcedir:
-	$(Q)mkdir -p $(SOURCEDIR)
-	$(Q)rsync -rt --exclude=$(BUILDDIR) . $(SOURCEDIR)
-
-doxy: copy-to-sourcedir
+# This is where we tweak the "pre" API comments into "preconditions"
+# Note that this step starts with an empty doc.log while the others append
+doxy:
 	$(Q)(cat acrn.doxyfile ; echo "OUTPUT_DIRECTORY=$(SOURCEDIR)/doxygen" ) | doxygen - > $(BUILDDIR)/doc.log 2>&1
 	$(Q)find $(SOURCEDIR)/doxygen/xml/* | xargs sed -i 's/simplesect kind="pre"/simplesect kind="preconditions"/'
 
-content: copy-to-sourcedir
+# Copy all the rst content (and images, etc) into the _build/rst folder
+# including rst content and xsd files from the /misc folder that we'll
+# use to generate config option documentation
+
+content:
+	$(Q)mkdir -p $(SOURCEDIR)
+	$(Q)rsync -rt --exclude=$(BUILDDIR) . $(SOURCEDIR)
 	$(Q)scripts/extract_content.py $(SOURCEDIR) misc
 	$(Q)mkdir -p $(SOURCEDIR)/misc/config_tools/schema
 	$(Q)rsync -rt ../misc/config_tools/schema/*.xsd $(SOURCEDIR)/misc/config_tools/schema
 	$(Q)xsltproc -xinclude ./scripts/configdoc.xsl $(SOURCEDIR)/misc/config_tools/schema/config.xsd > $(SOURCEDIR)/reference/configdoc.txt
 
+# Used to pull the acrn kernel source (for API docs)
+
 pullsource:
 	$(Q)scripts/pullsource.sh
 
-html: copy-to-sourcedir doxy content
+html: content doxy
+	@echo making HTML content
+	@./scripts/show-versions.py
```

Shouldn't we use
```suggestion
	$(Q)./scripts/show-versions.py
```
to be consistent with the rest of the Makefile?

---

yep to both